spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -85,4 +113,83 @@
     expect(page).to have_content("No bids yet.")
   end
 
+  scenario "Viewing bid history for running auction" do
+    Timecop.scale(36000) do
+      create_running_auction
+    end
+    path = auction_bids_path(@auction.id)
+    visit path
+
+    # sort the bids so that newest is first
+    bids = @auction.bids.sort_by {|bid| bid.created_at}.reverse
+
+    # ensure the table has the correct content, in the correct order
+    bids.each_with_index do |bid, i|
+      row_number = i + 1
+      unredacted_bidder_name = bid.bidder.name
+      bid = Presenter::Bid.new(bid)
+
+      # check the "name" column
+      within(:xpath, cel_xpath(row_number, 1)) do
```

Comment:
what about `expect_content_in_row_number(content: "[Name witheld until the auction ends]", row: 1)`

and defining 

```
def expect_content_in_row_number(args)
  within(:xpath, cel_xpath(row_number, args[:row_number])) do
     expect(page).to have_content(args[:content])
  end
end
```